CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## New Features:
 
-No changes to highlight.
+- New code component allows you to enter, edit and display code with full syntax highlighting by [@pngwn](https://github.com/pngwn) in [PR 3421](https://github.com/gradio-app/gradio/pull/3421)
```

Comment:
Could we add a screenshot here? I think lots of people would be interested in knowing how it looks!